allensdk/test/brain_observatory/behavior/data_objects/test_cell_specimens.py

```diff
@@ -28,7 +30,7 @@ def setup_class(cls):
             imaging_plane=ImagingPlane(
                 excitation_lambda=910.0,
                 indicator='GCaMP6f',
-                ophys_frame_rate=60.0,
+                ophys_frame_rate=10.0,
```

Comment:
Sorry I've been so out of the loop on this work, it's possible this has been discussed a lot already, but did we intend for this to change to a 10?

Reply:
I fixed issues with ophys timestamps in the PR, and this is the updated expected frame rate. It is correct given the new test timestamps.